packages/extension/package.json

```diff
@@ -841,6 +841,43 @@
                     "default": "prompt",
                     "description": "Default mode for results Open command."
                 },
+                "sqltools.csvExport.delimiter": {
+                    "type": "string",
+                    "minLength": 1,
+                    "default": ",",
+                    "description": "CSV field delimiter, one character only."
+                },
+                "sqltools.csvExport.header": {
+                    "type": "boolean",
+                    "default": true,
+                    "description": "Display the column names on the first line."
+                },
+                "sqltools.csvExport.quote": {
+                    "type": "string",
+                    "default": "\"",
+                    "description": "The quote characters, an empty value will preserve the original field."
+                },
+                "sqltools.csvExport.quoted": {
+                    "type": "boolean",
+                    "default": true,
+                    "description": "Quote all the non-empty fields even if not required."
+                },
+                "sqltools.csvExport.quoted_empty": {
+                    "type": "boolean",
+                    "default": true,
+                    "markdownDescription": "Quote empty fields and overrides `#sqltools.csvExport.quoted_string#` on empty strings when defined."
+                },
+                "sqltools.csvExport.quoted_string": {
+                    "type": "boolean",
+                    "default": false,
```

Comment:
Should this default to true so it aligns with what was previously passed to csvStringify?

Reply:
You're right. I swapped the default values for `quoted_string` and `quoted`.